src/script/main/app.ts

```diff
@@ -361,6 +361,8 @@ export class App {
       try {
         await this.core.init(clientType);
       } catch (error) {
+        const errorMessage = error instanceof Error ? error.message : error;
+        console.error(`Error when initializing core: "${errorMessage}"`, error);
```

Comment:
Shouldn't we go with `logger.error` instead?

Reply:
yes, we should 😅